ddtrace/tracer/payload.go

```diff
@@ -36,7 +36,7 @@ type payload struct {
 	off int
 
 	// count specifies the number of items in the stream.
-	count uint64
+	count atomic.Uint64
```

Comment:
I'm not familiar with how this is used, but from a glance I see that a new `payload` is created every so often when traces are flushed. Would this count ever realistically be larger than 2^32?

Reply:
There's no way this can ever get larger than uint32. We have a limit on the number of spans in a trace and the size of a payload that would make overflowing this impossible.